packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -1347,15 +1446,26 @@ class FollowerLayer extends ContainerLayer {
     _inverseDirty = true;
   }
 
+  /// {@template flutter.leaderFollower.markClean}
```

Comment:
if you keep these docs:

docs should have a leading paragraph that can be used as a one-line summary to show in API doc tables of contents.
docs should be in the passive style ("This method intentionally does..." rather than "We intentionally make this method do...").